tests/test_trajectory.py

```diff
@@ -627,7 +628,13 @@ def test_length(get_fn):
 
     for file in files:
         opened = md.open(get_fn(file))
-        loaded = md.load(get_fn(file), top=get_fn('native.pdb'))
+
+        if '.' + file.split('.', 1)[-1] in mdtraj.core.trajectory._TOPOLOGY_EXTS:
```

Comment:
best practice to get the last extension, feel free to ignore 
```suggestion
        if '.' + file.rsplit('.', 1)[-1] in mdtraj.core.trajectory._TOPOLOGY_EXTS:
```